lstchain/tools/lstchain_create_calibration_file.py

```diff
@@ -127,12 +125,7 @@ def setup(self):
 
             if self.eventsource.r0_r1_calibrator.drs4_pedestal_path.tel[tel_id] is None:
                 raise IOError("Missing (mandatory) drs4 pedestal file in trailets")
-
-            # remember how many events in the files
-            self.tot_events = len(self.eventsource.multi_file)
```

Comment:
not needed anymore?

Reply:
I took it out because it was no more calculated with the new multifile reader, and honestly with the reading of more subruns it is no more necessary (we will now discard the very last calibration sample which has in any case a lower statistics, hence it is better not to use it)

Reply:
but the field will be empty, right?

Reply:
`self.tot_events`

Reply:
this filed is not more there actually